### ISSUE TITLE
Performance of pre-trained model

### ISSUE THREAD

**Issue by leee222** [jgc128/mednli#2]
Hi

What is performance of the pre-trained model? is it the same model which achieve 73.5 accuracy on test dataset (Table 3 in Romanov and Shivade, EMNLP18 paper)? It will be helpful if you add the performance in the table in the below screenshot.

![image](https://user-images.githubusercontent.com/38415591/54802882-2629c880-4ca8-11e9-97f8-a291f9133122.png)

Also, when do you think you can share the more pre-trained models?

Thanks :)

**Comment by jgc128** [comment]
Hi,

The performance should be close to the one reported in the paper, although it may be a little bit lower or higher, since in the paper we report an average of several runs

We also provide pre-trained models of a simplified baseline here: https://github.com/jgc128/mednli_baseline#models - I'd suggest to use them as it is much simpler to use

**Comment by leee222** [comment]
Thanks for the answers
